src/cp_control_types.F

```diff
@@ -479,7 +479,8 @@ MODULE cp_control_types
                                               auto_basis_aux_fit = 1, &
                                               auto_basis_lri_aux = 1, &
                                               auto_basis_ri_hxc = 1, &
-                                              auto_basis_ri_xas = 1
+                                              auto_basis_ri_xas = 1, &
+                                              auto_basis_ri_hfx
```

Comment:
Does that mean that there is no default size for the RI_HFX basis ?

Reply:
It's initialized in `read_dft_control` with the same value `1` as the other basis types, but for consistency I updated this.